lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -28,11 +28,14 @@ def define
         save_input_as :housing_benefit
 
         next_node do |response|
-          chosen_cap = chosen_cap
```

Comment:
It would be better if the git history was re-written so this line was never introduced in the first place.